src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy

```diff
@@ -435,7 +435,17 @@ class OpenShiftDSL implements Serializable {
         for (int i = 0; i < names.length; i++) {
             String name = names[i].trim();
             if (! name.isEmpty()) {
-                results.add(name);
+               // to simplify life, to acount for new -o=name output for openshift unique api objs
+               // with group suffixes like deploymentconfig.apps.openshift.io, and chop off the "domain"
```

Comment:
so you're going to turn "deploymentconfig.apps.openshift.io/mydc" into "deploymentconfig/mydc"

what happens when there's also a "deploymentconfig.extensions.k8s.io"?

Reply:
we'll have to see how the actual "kind" field is represented when we have those two types of deploymentconfigs and adjust accordingly

going in the reverse direction breaks backwards compatibility and is a much more invasive change

I'm willing to add a //TODO noting this possibility

Reply:
i think the Kind fields will both say "deploymentconfig" and it's the apiversion field that will provide the disambiguation.

It's probably (hopefully) unlikely enough that we can ignore it for now, but i think end-users/administrators adding their own resource types is going to be more command and someone will probably get cute and create "deploymentconfig.mycorp.com" because they don't like the openshift one, someday.  So as long as you think there will be something we can do at that point, leaving it as a todo is ok with me.

Reply:
ok good thanks ... yeah, added the //TODO, and want to wait and see on this one, should be able to address when needed (the answer probably will be adding the version field to the key used for projectList, but I'd rather wait until we have something real to test against then anticipate what would be needed).